src/models/book.js

```diff
@@ -18,16 +18,19 @@ module.exports = (sequelize, DataTypes) => {
     }
   }, {});
 
-  Book.associate = function({Loan}) {
-    Book.hasMany(Loan);
+//  Book.associate = function({Loan}) {
```

Comment:
It's OK to remove code completely! It doesn't need to stay here commented out :)